.ci/conformance/run_tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+echo
+echo "Testing started `date +'%d.%m.%y %H:%M:%S'`."
+echo
+
+file_algs=algorithms.txt
+
+while read ALGORITHM_NAME
+    do 
+        echo Testing $ALGORITHM_NAME
+        echo
+        source ./tools/${ALGORITHM_NAME}/download_test.sh &>> _log_downloads
```

Comment:
Think common download script should be used now? 
ci/conformance/download_tests.sh